File: temp/debug_fix.js
const fs = require('fs');
const path = 'D:/新课开发/地产/物业/7.业主信任与老龄化接受度-AI落地绕不开的人情关/授课PPT/slides/slide-45.js';
let content = fs.readFileSync(path, 'utf8');

console.log('Contains U+201C:', content.includes('\u201C'));
console.log('Contains literal 201C:', content.includes(String.fromCharCode(0x201C)));

// Try direct replacement
const test = content.split('\u201C').join('\u201C');
console.log('After split/join, contains backslash-u201C:', test.includes('\u201C'));
console.log('After split/join, still contains U+201C:', test.includes('\u201C'));

fs.writeFileSync(path, test);
console.log('Written');

// Verify
try {
  new Function(test);
  console.log('Syntax OK');
} catch(e) {
  console.log('Still broken:', e.message.substring(0, 60));
}

File: temp/debug_fix.py
content = open('D:/新课开发/变革管理/03-组织变革就绪度评估：这家企业现在适不适合做深水区变革/完整课程包/02-授课PPT/slides/slide-53.js', 'r', encoding='utf-8').read()
lines = content.split('\n')
line = lines[47]
print(f"Line 48: {repr(line[:100])}")

# Test my function on just this line
def fix_inner_quotes(content):
    result = []
    i = 0
    quote_char = None

    while i < len(content):
        c = content[i]

        if quote_char is None:
            if c == '"' or c == "'":
                quote_char = c
                result.append(c)
                i += 1
            else:
                result.append(c)
                i += 1
        else:
            if c == '\\':
                result.append(c)
                if i + 1 < len(content):
                    result.append(content[i + 1])
                    i += 2
                else:
                    i += 1
            elif c == quote_char:
                result.append(c)
                quote_char = None
                i += 1
            elif c == '"' and quote_char == '"':
                result.append('\u300c')  # 「
                i += 1
            elif c == "'" and quote_char == "'":
                result.append('\u300c')  # 「
                i += 1
            else:
                result.append(c)
                i += 1

    return ''.join(result)

fixed = fix_inner_quotes(line)
print(f"Fixed: {repr(fixed[:100])}")